play-services-core/src/main/AndroidManifest.xml

```diff
@@ -90,6 +90,11 @@
         android:description="@string/perm_provision_description"
         android:label="@string/perm_provision_label"
         android:protectionLevel="privileged|signature" />
+    <permission
+        android:name="com.google.android.gms.permission.CAR_SPEED"
```

Comment:
Which app/issue is fixed by adding the `CAR_SPEED` permission? What about `CAR_MILEAGE` and `CAR_FUEL` permissions?

Reply:
Fixed the Google Maps location permission authorization pop-up window when connecting to Android Auto. The window still pops up after authorization.